### DISCUSSION TITLE
why?

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1594] [Q&A]
Frankly speaking, I didn't expect such a setup from SimpleLogin developers. To take and destroy a feature that was already free is not good. They could have at least brought it back for old users and new users would have already registered with the changes. It's a shame, though.

Moreover, your service is becoming less and less user friendly. While your direct competitors are adding features and possibilities - you introduce limitations. Not only that you have paid subdomains, unlike the same Anonaddy, which gives one free subdomain, but you in turn demand money for the basic function. So now you have also reduced the number of possible domains for generating aliases. And it has become even less convenient.

The big question is:
Why?

**Comment by nguyenkims** [comment]
This is duplicated to this discussion https://github.com/simple-login/app/discussions/1589, please continue the discussion there.
